src/ui/components/map/providers/mapboxmapprovider.js

```diff
@@ -33,7 +33,11 @@ export default class MapBoxMapProvider extends MapProvider {
       onload: () => {
         this._isLoaded = true;
         mapboxgl.accessToken = this._apiKey;
-
+        mapboxgl.setRTLTextPlugin(
+          'https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-rtl-text/v0.2.3/mapbox-gl-rtl-text.js',
```

Comment:
Could we use https://www.npmjs.com/package/@mapbox/mapbox-gl-rtl-text?activeTab=dependents instead?

Reply:
actually eh probably not worth the hassle for the SDK